python/lbann/contrib/lc/launcher.py

```diff
@@ -79,6 +79,15 @@ def set_environment(key, default):
     # present in MVAPICH2-2.3rc2.
     set_environment('MV2_USE_RDMA_CM', 0)
 
+    # Optimizations for Tioga
+    if system in ('tioga'):
+        #set_environment('NCCL_SOCKET_IFNAME', 'hsi')
+        set_environment('MIOPEN_DEBUG_DISABLE_FIND_DB', '1')
+        set_environment('MIOPEN_DISABLE_CACHE', '1')
+        set_environment('NCCL_DEBUG', 'INFO')
+        set_environment('NCCL_DEBUG_SUBSYS', 'INIT')
```

Comment:
I don't think we want to keep these long-term.

Reply:
Good catch.